homeassistant/components/vicare/__init__.py

```diff
@@ -36,7 +37,7 @@
 def setup(hass, config):
     """Create the ViCare component."""
     conf = config[DOMAIN]
-    params = {"token_file": "/tmp/vicare_token.save"}
+    params = {"token_file": tempfile.gettempdir() + "/vicare_token.save"}
```

Comment:
From my point of view...the token should be saved in the configuration directory of HA and not `/tmp`. As far as I can tell, doesn't `PyViCare` care where the token file is stored. But I guess that @oischinger know more about the decision about the location.

Reply:
I agree temp is not the best place (it can be purged  at any time)

Reply:
It doesn't really matter where it's stored since it's only read when Home Assistant is started. If it doesn't exist a new token is created automatically at that location.